schemas/pug-lintrc-schema.json

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
Since https://github.com/pugjs/pug-lint/pull/121 was merged, the schema is now included in the pug-lint package, so I think we can update this to reference the schema within `node_modules/pug-lint` and avoid duplication.